src/scripts/main.js

```diff
@@ -1 +1,83 @@
 'use strict';
+
+const body = document.querySelector('body');
+const div = document.createElement('div');
+
+function addElement() {
+  body.append(div);
+  div.dataset.qa = 'notification';
+}
+
+function firstPromise() {
+  const resolver1 = (complete, error) => {
+    setTimeout(() => {
+      error();
+    }, 3000);
+
+    body.addEventListener('mousedown', (e) => {
+      if (e.which === 1) {
+        complete();
+      }
+    });
+  };
+
+  return new Promise(resolver1);
+}
+
+const first = firstPromise();
+
+first
+  .then(() => {
+    addElement();
+    div.classList.add('success');
+    div.textContent = 'First promise was resolved';
```

Comment:
```suggestion
    div.classList.add('success');
    div.textContent = 'First promise was resolved';
```
Чому б не передавати клас і текст прямо у функцію addElement і не дублювати код?